README.md

```diff
@@ -84,6 +84,17 @@ hardhat:
 
 Otherwise, it defaults to the default mainnet provider plugin. You can also specify a ``block_number``.
 
+Hardhat deloyments are disabled for forks for performance reasons.  If you want your contract deployments you run on your fork, you can set ``no_deploy`` to ``False`` in your config.
+
+```yaml
+hardhat:
+  fork:
+    ethereum:
+      mainnet:
+        upstream_provider: alchemy
+        no_deploy: false
```

Comment:
I was thinking about that before but I think the name `no_deploy` makes the most sense since it directly relates to the hardhat option `--no-deploy`.   Though we could ignore that relationship entirely and go with something descriptive like `hardhat_deployments: true`.  Your call.